### PR TITLE
Enhanced participant-screens

### PR DESCRIPTION
**Pull request by selinaXyy**
I have changed the #8 state of widgets, and added more features based on which specific page the user's on. (The color of 'About Us' app bar was chosen from thc theme color!)

### CODE REVIEW

File: lib/views/home/home_screen.dart [hunk 1]
```diff
@@ -5,7 +5,7 @@ import 'package:thc/models/user.dart';
 import 'package:thc/views/admin_portal/admin_portal.dart';
 import 'package:thc/views/widgets.dart';
 
-class HomeScreen extends StatelessWidget {
+class HomeScreen extends StatefulWidget {
```
Comment (nate-thegrate) [inline lib/views/home/home_screen.dart:8]:
```suggestion
class HomeScreen extends StatelessWidget {
```

This widget should still be stateless. (One way to tell them apart: stateless widgets have a `build()` method, and stateful widgets have a `createState()` method.)

File: lib/views/home/home_screen.dart [hunk 2]
```diff
@@ -16,31 +16,56 @@ class HomeScreen extends StatelessWidget {
...
+          onTap: (index) {
+            setState(() {
+              _currentIndex = index;
+            });
+          },
+          items: const [
+            BottomNavigationBarItem(label: "Home", icon: Icon(Icons.home)),
+            BottomNavigationBarItem(label: "About Us", icon: Icon(Icons.info)),
+          ],
+        ),
```
Comment (nate-thegrate) [inline lib/views/home/home_screen.dart:68]: This looks fantastic, nice work!

**Review by nate-thegrate — CHANGES_REQUESTED** [review]
Just one small change and we'll be good!

File: lib/views/home/home_screen.dart (hunk 1 above)
Comment (selinaXyy) [reply lib/views/home/home_screen.dart]: Thanks! Just fixed that!

**Review by nate-thegrate — APPROVED** [review]
Nice work on the stateful widget implementation!

**Pull request merged**